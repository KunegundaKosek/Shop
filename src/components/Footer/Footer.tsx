import React from 'react';
import classes from './Footer.module.scss';

const Footer = () => {
    return <footer>Kunegunda Kosek</footer>;
};

export default Footer;
