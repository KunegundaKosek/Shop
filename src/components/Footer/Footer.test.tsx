import React from 'react';
import { render, screen } from '@testing-library/react';
import Footer from './Footer';
import { act } from 'react-dom/test-utils';

describe('Footer Component', () => {
    test('renders footer element', () => {
        const { container } = render(<Footer />);
        const footerElement = container.querySelector('footer');
        expect(footerElement).toBeInTheDocument();
    });
    test('renders footer text', () => {
        render(<Footer />);
        const footerElement = screen.getByText('Kunegunda Kosek');
        expect(footerElement).toBeInTheDocument();
    });

    test('matches snapshot', () => {
        const { container } = render(<Footer />);
        expect(container).toMatchSnapshot();
    });
});
