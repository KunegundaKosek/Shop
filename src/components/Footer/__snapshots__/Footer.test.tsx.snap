// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component matches snapshot 1`] = `
<div>
  <footer>
    Kunegunda Kosek
  </footer>
</div>
`;
